MockerTests/MockerTests.swift

```diff
@@ -87,14 +87,14 @@ final class MockerTests: XCTestCase {
         ]).register()
 
         URLSession.shared.dataTask(with: originalURL) { (data, _, error) in
-            XCTAssert(error == nil)
+            XCTAssertNil(error)
             guard let data = data, let image: UIImage = UIImage(data: data) else {
-                XCTFail("Invalid data")
+                XCTFail("Invalid data \(String(describing: data))")
```

Comment:
<!--
  1 failure:  variable declared...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: MockerTests/MockerTests.swift;
  Line: 92;
-->
- :no_entry_sign: variable declared in 'guard' condition is not usable in its body

Reply:
You might want to fix this @basthomas

Reply:
Hm, seems like a false positive, kind of? `data` is used to construct the `image`. Will take a look.

Reply:
Oooh, now I see. It's complaining about me using the "unavailable" variable within the body. "of course", this is just the nullable `data` we want to use. What do you think? Should I provide SwiftLint exceptions for these?

Reply:
I would either split the guard so data is available, or not log data at all in the exception I think!

Reply:
Hm, will split then 👍